src/stylesheets/theme/_uswds-theme-components.scss

```diff
@@ -25,7 +25,7 @@ $theme-accordion-font-family: "body";
 // Alert
 $theme-alert-bar-width: 1;
 $theme-alert-font-family: "ui";
-$theme-alert-icon-size: 4;
+$theme-alert-icon-size: 5;
```

Comment:
Check me on this one. This variable _is_ used in one spot, which I imagine would be affected by this change:

https://github.com/uswds/uswds/blob/480fa3464b428ecb0aafd4f27df237b2a4ff0dd9/src/stylesheets/core/mixins/_alert-status-styles.scss#L18